db/db_mysql/migrations/20200116000000_0.9.0_imap.sql

```diff
@@ -1,7 +1,7 @@
 
 -- +goose Up
 -- SQL in section 'Up' is executed when this migration is applied
-CREATE TABLE IF NOT EXISTS `imap` (user_id bigint,host varchar(255),port int,username varchar(255),password varchar(255),modified_date datetime,tls boolean,enabled boolean,folder varchar(255),restrict_domain varchar(255),delete_reported_campaign_email boolean,last_login datetime,imap_freq int);
+CREATE TABLE IF NOT EXISTS `imap` (user_id bigint,host varchar(255),port int,username varchar(255),password varchar(255),modified_date datetime,tls boolean,ignore_cert_errors boolean,enabled boolean,folder varchar(255),restrict_domain varchar(255),delete_reported_campaign_email boolean,last_login datetime,imap_freq int);
```

Comment:
Can we create a new migration for these?

Reply:
Ahhh yes of course. Fixed, I think.

Reply:
It looks like this isn't fixed. Ideally, we would just have two brand new migration files - one for MySQL and one for SQLite. Both of these would just alter the existing table.

Right now, we have an `ALTER` migration for MySQL, but this migration here still exists, and the one for SQLite still does a full `CREATE TABLE` instead of just altering the existing one.

Reply:
Apologies Jordan, I definitely made the alterations as you describe, I think I must have botched my stashes and commits. Pushed again 🤞